packages/react/src/components/TreeView/index.ts

```diff
@@ -8,7 +8,5 @@
 import TreeView from './TreeView';
 import TreeNode from './TreeNode';
 
-TreeView.TreeNode = TreeNode;
-
```

Comment:
I think we'll need to keep this in to avoid causing a breaking change for anyone using `<TreeView.TreeNode>`